lib/android/src/main/java/com/airbnb/android/react/maps/AirMapPolygon.java

```diff
@@ -41,6 +44,41 @@ public void setCoordinates(ReadableArray coordinates) {
         }
     }
 
+    public void setHoles(ReadableArray holes) {
+        // Convert ReadableArray<ReadableArray<ReadableMap>> into List<List<LatLng>>
+        int holesSize = holes.size();
+        this.holes = new ArrayList(holesSize);
+        for (int i = 0; i < holesSize; i++) {
+            ReadableArray hole = holes.getArray(i);
+            if(hole.size() > 3) {
+                List<LatLng> iterableHole = new ArrayList(hole.size());
+                this.holes.add(iterableHole);
+
+                if(BuildConfig.DEBUG){
+                    ReadableMap coordinate0 = hole.getMap(0);
+                    double lat0 = coordinate0.getDouble("latitude");
+                    double lon0 = coordinate0.getDouble("longitude");
+
+                    ReadableMap coordinate1 = hole.getMap(hole.size()-1);
+                    double lat1 = coordinate1.getDouble("latitude");
+                    double lon1 = coordinate1.getDouble("longitude");
+
+                    if(lat0 != lat1 || lon0 != lon1) {
+                        Log.e(TAG, "First and Last coordinates of the holes must be same");
```

Comment:
Because the hole will fail silently. No need to do these checks if we can avoid it.